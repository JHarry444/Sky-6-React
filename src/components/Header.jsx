function Header() {
    return (
        <header>
            <h1>My React App</h1>
            <h2>Is awesome</h2>
        </header>
    )
}
// makes the component available to other files
export default Header;