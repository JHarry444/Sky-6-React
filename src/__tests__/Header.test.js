import {create} from "react-test-renderer";
import Header from "../components/Header";
// Test class -> contains your tests
describe("snapshot testing", () => {
    // Test method -> runs the test
    it("should match the snapshot", function() {
        const header = create(<Header/>);
        expect(header.toJSON()).toMatchSnapshot();
    });
});