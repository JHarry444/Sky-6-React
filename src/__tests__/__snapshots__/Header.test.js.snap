// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing should match the snapshot 1`] = `
<header>
  <h1>
    My React App
  </h1>
  <h2>
    Is awesome
  </h2>
</header>
`;
